recipe/meta.yaml

```diff
@@ -1,58 +1,39 @@
 {% set version = "1.2.0" %}
 
-{% set variant = "openblas" %}
-
-
 package:
   name: cvxopt
   version: {{ version }}
 
 source:
-  fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
   sha256: 3296c9d49b7dcb894b20db5d7d1c1a443912b4d82358e03f836575e8398e0d60
-  patches: cvxopt_win_patch.diff   # [win]
-
 build:
-  skip: true  # [win and py<35]
-  number: 203
-  features:
-    - blas_{{ variant }}
+  number: 204
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - {{ compiler("c") }}
+    - c99-to-c89          # [win and vc<14]
   host:
     - python
     - setuptools
-    - blas 1.1 {{ variant }}
-    - openblas
+    - mkl-devel  {{ mkl }}  # [blas_impl == "mkl"]
+    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
     - gsl
     - fftw
-    - dsdp
     - glpk
-    - suitesparse 4.5.6|4.5.6.*   # [not win]
-    - suitesparse 5.1.*   # [win]
-
+    - dsdp
+    - suitesparse    # [unix]
   run:
     - python
-    - blas 1.1 {{ variant }}
-    - openblas
-    - gsl
-    - fftw
     - dsdp
-    - glpk
-    - suitesparse 4.5.6|4.5.6.*   # [not win]
-    - suitesparse 5.1.*   # [win]
 
 test:
   imports:
     - cvxopt
   source_files:
-    - tests/*
+    - tests/*.py
```

Comment:
At least one test depends on a data file (`boeing2.mps`) in the `tests` directory.

Reply:
I don't get how the AnacondaRecipes passes the tests :(